tests/spdx.rs

```diff
@@ -0,0 +1,27 @@
+extern crate license_exprs;
```

Comment:
Good call moving these to the tests directory! They are tests of the public API, they belong here and not inside the module.

Could you also include a test for a paren nested inside a larger expression, as in `(MIT AND MIT) OR MIT`?